sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaRegistryClient.cs

```diff
@@ -68,22 +70,14 @@ internal SchemaRegistryClient(ClientDiagnostics clientDiagnostics, HttpPipeline
         /// <param name="schemaContent">The string representation of the schema's content.</param>
         /// <param name="cancellationToken">The cancellation token for the operation.</param>
         /// <returns>The properties of the schema.</returns>
-        public virtual async Task<Response<SchemaProperties>> RegisterSchemaAsync(string groupName, string schemaName, SerializationType serializationType, string schemaContent, CancellationToken cancellationToken = default)
-        {
-            using DiagnosticScope scope = _clientDiagnostics.CreateScope(RegisterSchemaScopeName);
-            scope.Start();
-            try
-            {
-                var response = await RestClient.RegisterAsync(groupName, schemaName, serializationType, schemaContent, cancellationToken).ConfigureAwait(false);
-                var properties = new SchemaProperties(schemaContent, response.Headers.Location, response.Headers.SerializationType, response.Headers.SchemaId, response.Headers.SchemaVersion);
-                return Response.FromValue(properties, response);
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
-        }
+        public virtual async Task<Response<SchemaProperties>> RegisterSchemaAsync(
+            string groupName,
+            string schemaName,
+            SerializationType serializationType,
+            string schemaContent,
+            CancellationToken cancellationToken = default) =>
+            await RegisterSchemaInternalAsync(groupName, schemaName, serializationType, schemaContent, true, cancellationToken)
```

Comment:
```suggestion
                await RegisterSchemaInternalAsync(groupName, schemaName, serializationType, schemaContent, true, cancellationToken)
```

Reply:
It took me a moment to realize this wasn't just another parameter for the method.    I wasn't sure if this was intended to be aligned this way.

Reply:
Interesting - I always use this format for expression-bodied members. I haven't seen a convention here. If it really bothers you I can change it 😄

Reply:
I suppose putting the => on the next line would give a natural indentation.